test/unit/ajax.js

```diff
@@ -1590,6 +1592,19 @@ module( "ajax", {
 				ok( false, "error" );
 			}
 		} );
+
+		if ( jQuery.support.cors ) {
+			ajaxTest( "gh-1684 - jQuery.ajax() - support cross-domain PATCH in IE10-11", 1, {
+				url: "http://httpbin.org/patch",
```

Comment:
> If, for example, Spartan browser wouldn't have a `documentMode` property which is a likely scenario, your tests will not caught anything. Since they are to specific and ua-oriented.

I don't understand. Spartan doesn't have `document.documentMode` and doesn't have `window.ActiveXObject`. It will be treated as every non-IE browser so I don't see a problem here.

What's UA-oriented here besides the `documentMode` check that's only done to omit the test in non-IE browsers because of BrowserStack problems as it's obvious it works there as we're just using the native XHR always?